op-batcher/batcher/driver.go

```diff
@@ -195,6 +198,11 @@ func (l *BatchSubmitter) loadBlocksIntoState(ctx context.Context) {
 		return
 	}
 
+	end.Number -= l.SubUnsafeHeadNumbe
```

Comment:
It may be a potential bug: when `end.Number` is `uint64(1)` and `l.SubUnsafeHeadNumber` is `2`, then `uint64(1) - 2` will get a wrong result.

Reply:
will return at below statement.